renaud.mk

```diff
@@ -1,4 +1,4 @@
 INSTANCE_ID = renaud
 VARS_FILE = vars_renaud.yaml
-
+PRINT_OUTPUT = /tmp
```

Comment:
Note that using `/tmp` does not work as initially expected, because it leads the Makefile rule to do a mv command with the dest file being the same as the source file, which produces an error. We need to find another way.